### ISSUE TITLE
Cluster Monitoring Configuration

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#17734]
Version: Master 2.2 (1/29/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Proposal:**
The Cluster Monitoring Configuration page has a few issues:

- [x] When we enable cluster monitoring the fields (Data Retention period, node exporter host port, etc) should not disappear but should change to read-only (gray out).

- [x] The save button should change to edit once we have enabled monitoring so we can edit our settings here.

- [ ] Disable option should be next to the Save/Edit button.

**Result:**
This will bring this area more in alignment to how we have other pages set up in the Rancher UI.
This will make it easier to configure. We should not have to go to the "Update configuration" link under system project catalog apps to update configuration. Everything should be in one spot on this page only.

**Comment by cjellick** [comment]
i think this a bug, not an enhancement. updating type

**Comment by loganhz** [comment]
Maybe we can create another issue to talk about`Disable option should be next to the Save/Edit button.`?
It is by design. It's designed by UX.

**Comment by davidnuzik** [comment]
Thanks @loganhz if it's by UX design I'm okay with keeping it as it is.

**Comment by davidnuzik** [comment]
Version: Master 2.2 (2/20/19)
Confirmed enhancement. We can now edit cluster configuration and save any changes as needed.
I originally asked for an edit button but asked to change this back and I think it's fine the way it is now just as a save button. (I had thought we would have to gray out fields and click edit to enable editing then use save button to save).

I like the way that this is set up from a user-experience perspective. Thank you.

I smoke tested this area, saved changes to make sure they took effect, inspect http, etc. I did not identify any issues with my testing.

Here is what the Monitoring page looks like by default now after enabling monitoring.
![image](https://user-images.githubusercontent.com/45179589/53128232-9f0af880-3521-11e9-973a-cd5fb31710a7.png)
